cassandra-diagnostics-connector21/README.md

```diff
@@ -0,0 +1,51 @@
+# Cassandra Diagnostics Connector for Cassandra 2.1.x
+
+Connector is a module which hooks into the query path and extract information for diagnostics. Bytecode instrumentation is used to augment existing Cassandra code with addition functionality. It uses low priority threads to execute the diagnostics information extraction with minimal performance impact to the target code (Cassandra node or application/driver).
+
+## Tracing Slow Queries
+
+Tracing of slow queries enables watching and logging the actual queries which took longer to execute based on the configured treshold. The treshold is configured in Slow Query Module. This tracing is, then, output into the reporters and can be furthur processed or displayed on visualization tools' dashboards.
```

Comment:
- watching and logging => maybe only reporting or detecting
- This tracing is, then, output into the reporters => These slow queries are then reported via configured reporters...